example/pkg/apis/miskatonic/v1beta1/scale_university_types.go

```diff
@@ -18,53 +18,56 @@ package v1beta1
 
 import (
 	"context"
-	"github.com/kubernetes-incubator/apiserver-builder-alpha/example/pkg/apis/miskatonic"
 
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apiserver/pkg/registry/rest"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/kubernetes-incubator/apiserver-builder-alpha/example/pkg/apis/miskatonic"
 )
 
 // +genclient
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 
 // +subresource-request
-type Scale struct {
+type UniversityScale struct {
```

Comment:
do you mind updating the example here s/Scale/Status/? Scale is an exception, we should avoid that in the example. or i will fix it in another follow-up

Reply:
`status` is a common subresource that pinned in many resources. So I replace it with `refresh` to make it more typical.

Reply:
sorry for being dense, what does "UniversityRefresh" mean? wouldn't sth like "UniversityCampus", "UniversityCourse" better?

Reply:
well. "UniversityRefresh" is a bad example here. "UniversityCampus" is better. Done